idea/src/org/jetbrains/kotlin/idea/intentions/ConvertSealedSubClassToObjectIntention.kt

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2010-2017 JetBrains s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jetbrains.kotlin.idea.intentions
+
+import com.intellij.lang.Language
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.util.TextRange
+import com.intellij.psi.JavaPsiFacade
+import com.intellij.psi.PsiElement
+import com.intellij.psi.impl.source.tree.JavaElementType
+import com.intellij.psi.search.searches.ReferencesSearch
+import org.jetbrains.kotlin.KtNodeTypes
+import org.jetbrains.kotlin.lexer.KtTokens
+import org.jetbrains.kotlin.psi.*
+import org.jetbrains.kotlin.psi.psiUtil.getParentOfType
+
+class ConvertSealedSubClassToObjectIntention :
```

Comment:
Hello,

Tried to do it now. Now the tests don't work anymore because it seems that problems that extends LocalQuickFix doesn't appears on the actions list on tests. My test returned:

```
junit.framework.AssertionFailedError: Action with text 'Convert Sealed Sub-class 'SubSeal' to Object' is not available in test InterLanguage
Available actions (12): 
Create test
Edit intention settings
Disable 'Create test'
Make internal
Edit intention settings
Disable 'Make internal'
Make private
Edit intention settings
Disable 'Make private'
Move 'SubSeal' to separate file
Edit intention settings
Disable 'Move declaration to separate file'
```

On my tests on the IDE they all appear:
![screenshot from 2017-11-02 11-37-45](https://user-images.githubusercontent.com/7376453/32328928-9ad492ba-bfc2-11e7-8daa-9fe0c4c20315.png)

But it seems that the tests in the quickfix folder don't take them in consideration. Is there any simple way for me to test this?

Reply:
It's better to use here tests from `inspectionsLocal` directory (LocalInspectionsTestGenerated).

Reply:
Sorry, but I didn't find how to do multifile tests at `inspectionLocal`. I think there is no way to do that, is there?

I need to test changes in Kotlin files that caused changes in Java files. I think this is an important test.

Reply:
Sorry, I missed your message. You can use `QuickFixMultiFileTestGenerated`.